src/pages/login/login.js

```diff
@@ -32,8 +34,24 @@ export const Login=()=>{
         }
     }
 
+    const testLogin= async()=>{
+        try {
+            const response=await axios.post('/api/auth/login',{
+                email:"adarshbalika@gmail.com",
+                password:"adarshbalika"
+            })
+            localStorage.setItem("user",response.data.encodedToken);
+            setAuth(true)
+            navigate("/")
+        } catch (error) {
+            console.log(error);
+        }
+    }
+
```

Comment:
Instead of test login i would suggest something like `use test credentials` which will set the input values to the test user. This will avoid repetition of login logic.

Reply:
ok